src/gc/gc.d

```diff
@@ -699,9 +703,9 @@ struct GC
 
 
     /**
-     * Attempt to in-place enlarge the memory block pointed to by p by at least
-     * minbytes beyond its current capacity, up to a maximum of maxsize.  This
-     * does not attempt to move the memory block (like realloc() does).
+     * Attempt to in-place enlarge the memory block pointed to by p to at least
+     * minsize bytes, up to a maximum of maxsize.
+     * This does not attempt to move the memory block (like realloc() does).
```

Comment:
I don't like the current semantics too much, but extend is actually adding memory in the range minsize to maxsize. So the changed comment is incorrect.